extensions/microsoft-authentication/src/AADHelper.ts

```diff
@@ -71,6 +72,12 @@ export interface ITokenResponse {
 	id_token?: string;
 }
 
+type MicrosoftTokens = [accessToken: string, idToken?: string];
```

Comment:
Would prefer making this an object instead of an array

Reply:
Resolved, thanks!